addons/web_editor/models/ir_ui_view.py

```diff
@@ -224,6 +224,13 @@ def replace_arch_section(self, section_xpath, replacement, replace_tail=False):
         if not section_xpath:
             root = arch
         else:
+            if 'inherit_id' in arch.attrib:
+                inherit_view_ref = arch.attrib['inherit_id']
+                inherited_view = self.env.ref(inherit_view_ref)
+                inherited_arch = etree.fromstring(inherited_view.arch.encode('utf-8'))
+                for loc in arch.xpath(".//*[text()='$0']"):
+                    loc.text = ''
+                    loc.append(copy.deepcopy(inherited_arch[0]))
```

Comment:
@SergeBayet We can speak about your PR IRL if you prefer but I don't think this is the right solution. We should not fix the consequence but the cause of the error: the fact that the generated DOM has `/data/xpath[3]/form/t[1]/div[1]/button[1]/i[1]` is wrong in the first place IMO as it indicates that there is a editable `i` at the end of that path, in the view mentioned via `data-oe-model`, `data-oe-id`, which I think is not the case as it "jumps" views through `$0`? But let's talk about it IRL, maybe I don't remember this right.